frontend/model/contracts/group.js

```diff
@@ -1254,7 +1254,14 @@ sbp('chelonia/defineContract', {
           const sendingData = data.leavingGroup
             ? { member: data.member }
             : { member: data.member, username: meta.username }
-          await sbp('gi.actions/chatroom/leave', { contractID: data.chatRoomID, data: sendingData })
+          await sbp('gi.actions/chatroom/leave', {
+            contractID: data.chatRoomID,
+            data: sendingData,
+            ...(data.leavingGroup && {
+              signingKeyId: sbp('chelonia/contract/currentKeyIdByName', state, 'csk'),
+              innerSigningKeyId: null
+            })
```

Comment:
It's a bit confusing and unclear to someone reading this code for the first time why this condition is here and why this other CSK is being used. Can you please add a comment there?

Reply:
I've added the comment